packages/dicomImageLoader/src/shared/getTypedArrayFromMinMax.ts

```diff
@@ -0,0 +1,23 @@
+export default function getTypedArrayFromMinMax(min, max) {
```

Comment:
Shouldn't the type of this by:
getPixelDataTypeFromMinMax(min: number, max: number): (() => TypedArray)) { ....
I think that will declare it as a funtion returning a constructor for TypedArray